src/extension.ts

```diff
@@ -221,6 +229,11 @@ class ExtensionManager implements vscode.Disposable {
     return inst;
   }
 
+  private showToast: Promise<boolean | undefined> | undefined;
+  public async expShowToast(): Promise<boolean | undefined> {
```

Comment:
Because we don't wait for the exp service here, this might return undefined which is not awaitable.  I think we should add a variable to hold the exp service promise and wait for that when this.showToast is undefined.

Reply:
Actually, it looks like the variable exists in the telemetry module and we just have to call the method to get the exp service to access it.